plugin/pkg/scheduler/algorithm/priorities/selector_spreading_test.go

```diff
@@ -228,6 +229,203 @@ func TestSelectorSpreadPriority(t *testing.T) {
 	}
 }
 
+func TestZoneSelectorSpreadPriority(t *testing.T) {
+	labels1 := map[string]string{
+		"label1": "l1",
+		"baz":    "blah",
+	}
+	labels2 := map[string]string{
+		"label2": "l2",
+		"baz":    "blah",
+	}
+	nodeLabelsZone1 := map[string]string{
+		wellknownlabels.LabelZoneFailureDomain: "zone1",
+	}
+	nodeLabelsZone2 := map[string]string{
+		wellknownlabels.LabelZoneFailureDomain: "zone2",
+	}
+	nodeLabelsZone3 := map[string]string{
+		wellknownlabels.LabelZoneFailureDomain: "zone3",
+	}
+	labeledNodes := map[string]map[string]string{
+		"machine1.zone1": nodeLabelsZone1,
```

Comment:
I'm a bit worried about how many times these string literals are cut'n pasted throughout this source file.  A single constant array, or a generator function would significantly reduce the probability of these drifting out of sync.